DOCUMENTATION.md

```diff
@@ -553,18 +553,22 @@ In the following chapters you will learn the description of all options supporte
 
 Serialization is the process of converting an object into a stream of bytes to store the object in the memory, a file or database, or transmit it through the network. Its main purpose is to save the state of an object in order to be able to recreate it when needed. The reverse process is called deserialization. Hazelcast offers you its own native serialization methods. You will see these methods throughout this chapter.
 
-Hazelcast serializes all your objects before sending them to the server. The `boolean`, `number`,`string` and `Long` types are serialized natively and you cannot override this behavior. The following table is the conversion of types for the Java server side.
+Hazelcast serializes all your objects before sending them to the server. Certain types, like `boolean`, `number`, `string`, and `Long`, are serialized natively and you cannot override this behavior. The following table is the conversion of types for the Java server side.
 
-| Node.js | Java                                |
-|---------|-------------------------------------|
-| boolean | Boolean                             |
-| number  | Byte, Short, Integer, Float, Double |
-| string  | String                              |
-| Long    | Long                                |
+| Node.js         | Java                                  |
+|-----------------|---------------------------------------|
+| boolean         | Boolean                               |
+| number          | Byte, Short, Integer, Float, Double   |
+| string          | String                                |
+| Long            | Long                                  |
+| Buffer          | byte[]                                |
+| Object          | com.hazelcast.core.HazelcastJsonValue |
```

Comment:
Should we also put `HazelcastJsonValue` (the client's class) here?

Reply:
It's described in the `JSON Serialization` section and most users won't have to deal with it. Only those who want to avoid eager JSON parsing will set `jsonStringDeserializationPolicy` to `NO_DESERIALIZATION` and then work with instances of the class.

So, I've decided to keep the table as small as possible. We also have `RestValue` which I didn't mention in the table for the same reason.

WDYT?

Reply:
Okay then, it seems reasonable